.krew.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubescape
+spec:
```

Comment:
@Oshratn Please review the `shortDescription`, `description`, and `homepage` content

Reply:
LGTM

Reply:
if this is a project integration it should be kubescape.io (which is a link to GitHub until it's a real site)